cookbook/src/components/MainPage/style/MainPageComponentStyle.js

```diff
@@ -0,0 +1,88 @@
+import styled, { css } from 'styled-components';
+import { ButtonStyled, Container, H1Styled, ParagraphStyled } from '../../common/StylesComponent';
```

Comment:
H1Styled -> Heading or Title

Reply:
Why the bad practise is only H1Styled?
What if in future i'm gonna create styled for h2, h3?

Reply:
Because `H1Styled` doesn't show the idea/purpose of this component.
Let's use props to configure component.
Example:

```
<Heading size="small">some text</Heading>
<Heading size="medium">some text</Heading> // you can make size="medium" as default size
<Heading size="big">some text</Heading>

```